trollflow2/launcher.py

```diff
@@ -266,7 +266,7 @@ def _extract_filenames(msg):
 
     If the message contains a `filesystem` item, use fsspec to decode it.
     """
-    filenames = [urlparse(uri).path for uri in gen_dict_extract(msg.data, 'uri')]
+    filenames = [uri.split("::")[0] for uri in gen_dict_extract(msg.data, 'uri')]
```

Comment:
Wouldn't this return just `'simplecache'` if the URI is something like `simplecache::s3://noaa-goes17/ABI-L1b-RadF/2021/195/20/*_G17_s20211952030*`?

Reply:
Seems reasonable @pnuu , perhaps we should add a test for that then?

Reply:
Looking at it now